cakephpsphinx/themes/cakephp/static/css/default.css

```diff
@@ -192,8 +192,8 @@
 }
 
 #sidebar-navigation > ul {
-  padding-left: 0;
-  margin: 0;
+  padding-left: 15px;
```

Comment:
Why add indentation? If we could fix the vertical spacing we wouldn't need indentation.